<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="5" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="277" started-at="2019-05-14T10:00:08Z" finished-at="2019-05-14T10:00:08Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="277" started-at="2019-05-14T10:00:08Z" finished-at="2019-05-14T10:00:08Z">
      <class name="com.test.api.TestNGClass.IdAuth">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="beforeSuite" is-config="true" duration-ms="18" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="beforeTest" is-config="true" duration-ms="1" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="beforeClass" is-config="true" duration-ms="0" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="baiduSearchTest()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="baiduSearchTest" duration-ms="93" started-at="2019-05-14T18:00:08Z" description="百度搜索" finished-at="2019-05-14T18:00:08Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.test.api.TestNGClass.IdAuth.baiduSearchTest(IdAuth.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- baiduSearchTest -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="afterMethod" is-config="true" duration-ms="0" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="f()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="f" duration-ms="62" started-at="2019-05-14T18:00:08Z" description="测试新增人员" finished-at="2019-05-14T18:00:08Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.test.api.TestNGClass.IdAuth.f(IdAuth.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="afterMethod" is-config="true" duration-ms="0" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="sougouSearchTest()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="sougouSearchTest" duration-ms="9" started-at="2019-05-14T18:00:08Z" description="搜狗搜索" finished-at="2019-05-14T18:00:08Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.test.api.TestNGClass.IdAuth.sougouSearchTest(IdAuth.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sougouSearchTest -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="afterMethod" is-config="true" duration-ms="0" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="w(java.lang.Integer, java.lang.String)[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="w" duration-ms="26" started-at="2019-05-14T18:00:08Z" data-provider="dp" finished-at="2019-05-14T18:00:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[a]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [1] but found [2]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [1] but found [2]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:167)
	at com.test.api.TestNGClass.IdAuth.w(IdAuth.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- w -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="afterMethod" is-config="true" duration-ms="1" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="w(java.lang.Integer, java.lang.String)[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="w" duration-ms="9" started-at="2019-05-14T18:00:08Z" data-provider="dp" finished-at="2019-05-14T18:00:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[b]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- w -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="afterMethod" is-config="true" duration-ms="0" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="afterClass" is-config="true" duration-ms="1" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="afterTest" is-config="true" duration-ms="0" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:com.test.api.TestNGClass.IdAuth@7243145f]" name="afterSuite" is-config="true" duration-ms="0" started-at="2019-05-14T18:00:08Z" finished-at="2019-05-14T18:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- com.test.api.TestNGClass.IdAuth -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
